mfglib/alg/fictitious_play.py

```diff
@@ -183,9 +183,15 @@ def solve(
 
         return solutions, scores, runtimes
 
-    @classmethod
-    def _init_tuner_instance(cls, trial: optuna.Trial) -> FictitiousPlay:
+    def _init_tuner_instance(self, trial: optuna.Trial) -> FictitiousPlay:
         alpha_bool = trial.suggest_categorical("alpha_bool", [False, True])
         alpha_num = trial.suggest_float("alpha_num", 0.0, 1.0)
         alpha = None if alpha_bool else alpha_num
         return FictitiousPlay(alpha=alpha)
+
+    def from_study(self, study: optuna.Study) -> FictitiousPlay:
+        best_params = study.best_params
+        alpha_bool = best_params.pop("alpha_bool")
+        alpha_num = best_params.pop("alpha_num")
+        alpha = None if alpha_bool else alpha_num
+        return FictitiousPlay(alpha=alpha)
```

Comment:
Currently `FictitiousPlay(alpha=0)` and `FictitiousPlay(alpha=None)` behave identically. I'd love to drop support for `alpha=None` and only support `alpha=0`. Then we could simplify the above as

```python
    def _init_tuner_instance(self, trial: optuna.Trial) -> FictitiousPlay:
        return FictitiousPlay(alpha=trial.suggest_float("alpha", 0.0, 1.0))
```

and rely on the superclass implementation of `from_study`.